hawkular-bus-common/src/main/java/org/hawkular/bus/common/SimpleMessageProcessor.java

```diff
@@ -67,16 +65,16 @@ public MessageId send(BasicMessage basicMessage, Map<String, String> headers) th
         return super.sendAndListen(producerCtx, basicMessage, responseListener, headers);
     }
 
-    public <R extends BasicMessage> ListenableFuture<BasicMessageWithExtraData<R>> sendRPC(BasicMessage basicMessage,
-            Class<R> expectedResponseMessageClass) throws JMSException {
-        return super.sendRPC(producerCtx, basicMessage, expectedResponseMessageClass);
-    }
-
-    public <R extends BasicMessage> ListenableFuture<BasicMessageWithExtraData<R>> sendRPC(BasicMessage basicMessage,
-            Class<R> expectedResponseMessageClass, Map<String, String> headers)
-            throws JMSException {
-        return super.sendRPC(producerCtx, basicMessage, expectedResponseMessageClass, headers);
-    }
+//    public <R extends BasicMessage> ListenableFuture<BasicMessageWithExtraData<R>> sendRPC(BasicMessage basicMessage,
```

Comment:
if we no longer want to support synchronous messaging (I actually thought it was a neat feature of the hawkular bus API, but I don't think anyone is using it today) then delete this entirely.